Formula/openjdk.rb

```diff
@@ -70,10 +70,10 @@ class Openjdk < Formula
     end
   end
 
-  # Calculate Xcode's dual-arch JavaNativeFoundation.framework path
+  # Path to dual-arch JavaNativeFoundation.framework from the CLT
+  # (not Xcode, since not all users have Xcode installed)
   def framework_path
-    File.expand_path("../SharedFrameworks/ContentDeliveryServices.framework/Versions/Current/itms/java/Frameworks",
-                     MacOS::Xcode.prefix)
+    MacOS.sdk_path/"System/Library/Frameworks/JavaNativeFoundation.framework"
```

Comment:
Don't you want the directory containing `JavaNativeFoundation.framework` here, and not the full path?